cve_bin_tool/checkers/luajit.py

```diff
@@ -0,0 +1,18 @@
+# Copyright (C) 2022 Orange
```

Comment:
Do you want a different copyright string?  I'm guessing "Orange" may not be sufficient to identify you.

Reply:
(Intel's licensing team prefers "Intel Corporation" so I wanted to double check in case Orange's licensing folk prefer "Orange Corporation" or something similar.)

Reply:
"Orange" is enough, that's what our IP team asks us to add in all our open source projects or contributions.